i18n.php

```diff
@@ -59,6 +59,21 @@
     'then' => 'اذاً|ثم',
     'when' => 'عندما|متى',
   ),
+  'az' => 
+  array (
+    'and' => 'Və/Həm',
```

Comment:
it is expected to have a ``/`` here ? shouldn't it be 2 alternatives (and so a ``|``) ?

But this allowed me to catch a bug in gherkin btw.